public/class-public.php

```diff
@@ -119,6 +119,7 @@ public function new_account_page()
       $menu_links = array_slice($menu_links, 0, 1, true)
         + $new_menu_links
         + array_slice($menu_links, 1, NULL, true);
+      $menu_links['orders'] = __('My Orders', 'textdomain');
```

Comment:
This should go in the Radical skincare theme or the `vitalibis-wp`. Why do you want it to go here?

Reply:
Oh at first i did it in Radical theme but then i thought it is related to this plugin thats why added it here. Let me move it in the radical.